apps/ais/src/assets/data/guidedTour.json

```diff
@@ -81,5 +81,11 @@
       "element": "#profile",
       "intro": "Wir wünschen dir viel Spaß beim Erkunden verschiedener Fundstücke."
     }
+  ],
+
+  "location": [
+    {
+      "url": "fundstuecke/tree/37"
```

Comment:
Ha! Es funktioniert ja anscheindend 😄 
Aber in ein JSON kann man auch einfach alles reinschreiben...